client/src/components/auth/Login.js

```diff
@@ -21,15 +21,23 @@ export default class Login extends Component {
   async handleSubmit(event) {
```

Comment:
if you're using `.then` notation in line 33, then you can remove the `async` keyword from function definition